extensions/amp-fx-flying-carpet/0.1/amp-fx-flying-carpet.js

```diff
@@ -110,15 +110,15 @@ class AmpFlyingCarpet extends AMP.BaseElement {
     const docHeight = viewport.getScrollHeight();
     // Hmm, can the page height change and affect us?
     user().assert(
-      layoutBox.top >= viewportHeight,
+      layoutBox.top >= viewportHeight * 0.8,
```

Comment:
@jridgewell I'm okay with this since the overlap and the box could be a max of 0.2 * viewport. Worst case will look like this, which still shows the bottom of the content on scroll. 

<img width="293" alt="screen shot 2017-03-21 at 8 52 19 pm" src="https://cloud.githubusercontent.com/assets/2366341/24181496/7b1aa41a-0e78-11e7-8bc1-95de280dfadb.png">

Missing anything? (btw, didn't find any documentation asking not to add flying carpet in the first and last viewport - did we just miss it?)